.travis.yml

```diff
@@ -44,7 +44,6 @@ after_failure:
       fi
 
 matrix:
-  fast_finish: true
```

Comment:
<bikeshedding>I'd leave `fast_finish` there, we will need it in the future, right?</>